Dockerfile.builder

```diff
@@ -14,7 +14,7 @@ WORKDIR /go/src/github.com/cilium/cilium
 ENV GOROOT /usr/local/go
 ENV GOPATH /go
 ENV PATH "$GOROOT/bin:$GOPATH/bin:$PATH"
-ENV GO_VERSION 1.11.5
+ENV GO_VERSION 1.12
```

Comment:
will `1.12` always resolve to `1.12.0` ?

Reply:
1.12.0 does not exist, only 1.12, when 1.12.1 is released then 1.12.1 will be added